store/src/java-test/com/zimbra/cs/service/account/GetInfoTest.java

```diff
@@ -0,0 +1,94 @@
+package com.zimbra.cs.service.account;
+
+import com.zimbra.cs.account.MockProvisioning;
+import com.zimbra.cs.account.Provisioning;
+import com.zimbra.cs.account.Server;
+import com.zimbra.cs.mailbox.MailboxTestUtil;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class GetInfoTest {
+
+    private static Server server = null;
+    private String[] servicesEnabled = null;
+    private String[] servicesInstalled = null;
+    private String[] spellCheckUrls = null;
```

Comment:
The three variables should not be member variables but be defined in each test case to keep dependency among tests.
Server settings should be set using `setAttributes(String[] servicesEnabled, String[] servicesInstalled, String[] urls)` in each test case for the same reason.

Reply:
updated